data/altered.go

```diff
@@ -0,0 +1,85 @@
+package data
+
+// AlteredAccount is a structure that holds information about an altered account
+type AlteredAccount struct {
+	IsSender        bool
+	IsESDTOperation bool
+	IsNFTOperation  bool
+	TokenIdentifier string
+	NFTNonceString  string
+}
+
+type AlteredAccounts struct {
+	altered map[string][]*AlteredAccount
```

Comment:
can this be called in a concurrent manner? There is no warning that states that this component is concurrent safe or not safe

Reply:
this method is not called in a concurrent manner. the worker behind the data indexer has a single goroutine.

Reply:
ok, can remain as it is for performance reasons